src/components/FlyoutMenu/_flyout-menu.scss

```diff
@@ -59,12 +59,12 @@ $shadow: 0.4rem;
       content: '';
       position: absolute;
       top: -$shadow-plus-padding;
-      left: -1rem;
+      left: var(--after-offset);
```

Comment:
This needs a default set for when the component isn't using the helper function and/or the conditional in flyout menu fails https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1417/files#diff-f98c4644941be7c8613bb9b77020bc7aR100